src/main/java/com/vkatit/cinema/controller/PdfController.java

```diff
@@ -0,0 +1,50 @@
+package com.vkatit.cinema.controller;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import com.vkatit.cinema.model.Ticket;
+import com.vkatit.cinema.service.Printable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/api/v1/pdf")
+public class PdfController {
+
+    private final Printable printable;
+
+    public PdfController(Printable printable) {
+        this.printable = printable;
+    }
+
+    @GetMapping("/download")
+    public ResponseEntity<?> download() {
+        int id = 123;
```

Comment:
should extract name from path and use it. all other fields and class ticket can be removed. In jira was said "Add api /pdf/ticketid.pdf that return bytes in image/pdf format" Nothing about ticket. ticketid.pdf is name of file. we assume file already exists and we expose it.

Reply:
Method extracts the filename from the path
Ticket class and fields have been removed